doc/examples/reconst_csd.py

```diff
@@ -160,7 +145,7 @@
 ren = fvtk.ren()
 
 fvtk.add(ren, response_actor)
-
+print('Saving illustration as csd_response.png')
 fvtk.record(ren, out_path='csd_recursive_response.png', size=(200, 200))
```

Comment:
That's what I get for copy-pasting.

On Sat, Mar 14, 2015 at 2:48 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In doc/examples/reconst_csd.py
> <https://github.com/MrBago/dipy/pull/14#discussion_r26442098>:
>
> > @@ -160,7 +145,7 @@
> >  ren = fvtk.ren()
> >
> >  fvtk.add(ren, response_actor)
> > -
> > +print('Saving illustration as csd_response.png')
> >  fvtk.record(ren, out_path='csd_recursive_response.png', size=(200, 200))
>
> Those two lines are contradicting themselves.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MrBago/dipy/pull/14/files#r26442098>.
>